### PR TITLE
feat: Dashboard now displaying name, specRef and weight in the experimental step.

### PR DESCRIPTION
**Pull request by schakrad**
Before:
Screen Shot 2022-02-14 at 5 48 19 PM

After:
Screen Shot 2022-02-18 at 12 35 56 PM
Screen Shot 2022-02-18 at 12 36 26 PM

Checklist:
* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-rollouts/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this is a chore.
* [ ] The title of the PR is (a) [conventional](https://www.conventionalcommits.org/en/v1.0.0/), (b) states what changed, and (c) suffixes the related issues number. E.g. `"fix(controller): Updates such and such. Fixes #1234"`.  
* [x] I've signed my commits with [DCO](https://github.com/argoproj/argoproj)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My builds are green. Try syncing with master if they are not. 
* [ ] My organization is added to [USERS.md](https://github.com/argoproj/argo-rollouts/blob/master/USERS.md).

### CODE REVIEW

**Comment by harikrongali** [comment]
fixes #1463

File: ui/src/app/components/rollout/rollout.tsx [hunk 1]
```diff
@@ -297,8 +297,23 @@ const Step = (props: {step: GithubComArgoprojArgoRolloutsPkgApisRolloutsV1alpha1
...
-                {unit}
+                <EffectDiv className={`steps__step-title ${props.step.experiment ? 'steps__step-title--experiment' : ''}`}>
+                    <i className={`fa ${icon}`} /> {content}
+                    {unit}
+                </EffectDiv>
+                {props.step.experiment?.templates && (
+                    <EffectDiv className='steps__step__content'>
+                        {props.step.experiment?.templates.map((template) => {
+                            return (
+                                <EffectDiv className='steps__step__content-body'>
```
Comment (rbreeze) [inline ui/src/app/components/rollout/rollout.tsx:308]: These can just be normal `div`s, not `EffectDiv`s. `EffectDiv` is for adding effects on hover, etc. which isn't needed here

File: ui/src/app/components/rollout/rollout.tsx [hunk 2]
```diff
@@ -297,8 +297,23 @@ const Step = (props: {step: GithubComArgoprojArgoRolloutsPkgApisRolloutsV1alpha1
...
+                                    <EffectDiv>name: {template.name}</EffectDiv>
```
Comment (rbreeze) [inline ui/src/app/components/rollout/rollout.tsx:309]: Could you place the labels and values on separate lines to better visually separate them? Maybe we could capitalize the labels as well, something like this: 

Screen Shot 2022-02-15 at 16 07 27

**Review by rbreeze — CHANGES_REQUESTED** [review]

File: ui/src/app/components/rollout/rollout.tsx (hunk 2 above)
Comment (schakrad) [reply ui/src/app/components/rollout/rollout.tsx:309]: Sure, will modify.

Comment (schakrad) [reply ui/src/app/components/rollout/rollout.tsx:309]: @rbreeze Each template consists of name, specRef and weight. An experiment can consist of many templates. How do you want to segregate the templates.

**Review by alexmt — APPROVED** [review]
LGTM

**Comment by alexmt** [comment]
@rbreeze , can you please take another look. I think your comments were addressed

**Review by rbreeze — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**